tools/update-prod-static

```diff
@@ -14,6 +14,7 @@ sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
 import scripts.lib.setup_path_on_import
 os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'
 from django.conf import settings
+from tools.provision import check_node_cache
```

Comment:
I would probably move the node cache management to its own file under `scripts.lib`